core/include/gnuradio-4.0/Graph.hpp

```diff
@@ -591,16 +613,28 @@ class Edge {
     }
 };
 
-struct Graph {
+class Graph : public gr::Block<Graph> {
```

Comment:
The move assignment operator now gives me a sonar warning `Move assignment operator of ‘Graph’ is implicitly deleted because base class gr::Block<Graph> has deleted move assignment operator.`. opendigitizer's GnuRadioAcquisitionWorker relies on this (moving around graphs in/out of an optional)

Reply:
Will see what can be done. If this can not be fixed (needs to see why Block is not moveable), GnuRadioAcquisitionWorker could use a unique_ptr<Graph> instead of optional.